internal/textfile/read_test.go

```diff
@@ -60,10 +68,15 @@ func TestRead(t *testing.T) {
 				want = test.data
 			}
 
-			f, cleanup := writeTempfile(t, test.data)
-			defer cleanup()
+			tempname := writeTempfile(t, test.data)
+			defer func() {
```

Comment:
I'd like to keep the cleanup function as a return value of `writeTempfile`, so that a user just has to call `defer cleanup()` as before

Reply:
Done.